src/io/iohk/scalanet/peergroup/InetMultiAddress.scala

```diff
@@ -0,0 +1,22 @@
+package io.iohk.scalanet.peergroup
+
+import java.net.{InetAddress, InetSocketAddress}
+
+case class InetMultiAddress(private[scalanet] val inetSocketAddress: InetSocketAddress) {
```

Comment:
A rationale for the introduction of `InetMultiAddress` would be useful.

Reply:
Added.